client/src/components/HomeScreen.tsx

```diff
@@ -208,9 +208,7 @@ const CreateAuditWrapper = styled.div`
   padding: 30px;
 `
 
-// TODO: remove "display: none;" when we implement Minerva
 const BallotPollingWrapper = styled.div`
-  display: none;
```

Comment:
I think for now we're going to do this a little differently (@benadida is working on a configuration change). So this will probably get a little more complex.

Reply:
Got it. Should I take it out then?

Reply:
I think it's probably fine to leave in.